examples/workflow-test/README.md

```diff
@@ -43,6 +43,14 @@ The following libraries/frameworks need to be installed on your system:
 -   [Node.js](https://nodejs.org/en/) `>=16.11.0`
 -   [Yarn](https://classic.yarnpkg.com/en/docs/install#debian-stable) `>=1.7.0`
 
+## Min versions
+
+-   [Standalone](https://github.com/eclipse-glsp/glsp-client): v2.1.1
```

Comment:
What we should add instead here is the minimum required VS Code version. I assume since the newest version changed the dom for the file explorer the tests no longer with older versions right?